src/Symfony/Component/Notifier/Bridge/Bluesky/Tests/BlueskyTransportTest.php

```diff
@@ -238,6 +238,39 @@ public function testParseFacetsUrlWithEmoji()
         $this->assertEquals($expected, $output);
     }
 
+    public function testParseFacetsHashTags()
+    {
+        $input = 'Salut #test #123 #テスト http://bsky.app';
```

Comment:
Maybe you could update the URL `http://bsky.app` to use something like `http://bsky.app#fragment` to see if the regex doesn't catch the URL fragment as a hashtag. Thanks.

Reply:
Regarding URLs (and tags), this is already the default behavior when not using facets. Any benefits in doing it via facets?